src/commands/a32nx/flex-temp.ts

```diff
@@ -0,0 +1,18 @@
+import { CommandDefinition } from '../../lib/command';
+import { CommandCategory } from '../../constants';
+import { makeEmbed } from '../../lib/embed';
+
+export const flexTemp: CommandDefinition = {
+    name: ['flextemp', 'flex'],
+    description: 'Provides a link to the a32nx flex temp guide',
+    category: CommandCategory.A32NX,
+    executor: async (msg) => {
+        const flexEmbed = makeEmbed({
+            title: 'FlyByWire A32NX | Flex Temp',
+            description: 'Please see our [guide](https://docs.flybywiresim.com/pilots-corner/beginner-guide/preparing-mcdu/#flex-temp) on flex temp for information on how to set it correctly.',
```

Comment:
```suggestion
            description: 'A320neo takeoff performance data is not readily available. To set your Flex Temp in the simulator please see the [Flex Temp Section](https://docs.flybywiresim.com/pilots-corner/beginner-guide/preparing-mcdu/#flex-temp) in our MCDU guide.
```

I'd say reword the first sentence if necessary but just trying to avoid "guide" here as it's a very short admonition and explanation of flex temp.